### PR TITLE
Fixes #1455 - prevent shortkeys to fire in comment textarea

### LINKED ISSUE #1455: Click "e" on webcompat.com comment area opens label menu
Expect behavior: nothing happen
Actual behavior: click "e" always open label menu, which interrupt comment.

### PR DESCRIPTION
**Pull request by zoepage**

### CODE REVIEW

File: webcompat/static/js/lib/labels.js [hunk 1]
```diff
@@ -24,9 +24,9 @@ issues.LabelsView = Backbone.View.extend({
     "click .js-LabelEditorLauncher.is-active": "closeEditor"
   },
   keyboardEvents: {
-    "l": "labelWrap"
+    "l": "labelWarp"
```
Comment (miketaylr) [inline webcompat/static/js/lib/labels.js]: Ah, now I get `labelWrap`, it was a typo for `labelWarp`. :)))

So, the name `githubWarp` was a silly name playing off the mario brothers concept of using a pipe to warp to another level:

 (hence the dumb `warpPipe` variable name...)

So I'm not sure `labelWarp` is meaningful. What we're doing is opening the label editor.

**Comment by miketaylr** [comment]
(I think this needs to be rebased against latest master...)

So, it turns out this regression was introduced by https://github.com/webcompat/webcompat.com/pull/1431... and it was my suggestion to use mousetrap to handle ctrl+enter... which meant the two existing shortcuts would work. Oops. 🙈 

This looks good, but we should rename the `labelWarp` to something else like `openEditor` or `editLabels`, IMO.

**Comment by miketaylr** [comment]
I'll file a followup issue for the renaming, that's a very good "good first bug". :)

File: webcompat/static/js/lib/labels.js (hunk 1 above)
Comment (zoepage) [inline webcompat/static/js/lib/labels.js]: I got the reference and liked the idea, but I also get the benefits of having a meaningful name. 
+1 on "good first bug"

**Comment by zoepage** [comment]
@miketaylr you merged, so no rebase? :)

**Comment by miketaylr** [comment]
> @miketaylr you merged, so no rebase? :)

Yeah, sorry -- the merge applied cleanly. All good!

**Pull request merged**